### ISSUE TITLE
Typo in compile_stop_callbacks/2 spec

### ISSUE THREAD

**Issue by wfgilman** [phoenixframework/phoenix#2163]
### Environment

* Phoenix version (mix deps): 1.3 rc

### Expected behavior

In the `Phoenix.Endpoint.Instrument` module, `compile_start_callbacks/2` and `compile_stop_callbacks/2` have the same spec.

https://github.com/phoenixframework/phoenix/blob/v1.3.0-rc.0/lib/phoenix/endpoint/instrument.ex#L160

```
@spec compile_stop_callbacks(term, [module]) :: Macro.t
def compile_stop_callbacks(event, instrumenters) do
```

### Actual behavior

```
@spec compile_start_callbacks(term, [module]) :: Macro.t
def compile_stop_callbacks(event, instrumenters) do
```

**Comment by josevalim** [comment]
Yes, can you please send a PR?

**Comment by wfgilman** [comment]
Fixed with #2164
